REPORT/makefile

```
@@ -250,7 +257,7 @@ es-template: es-check
 	echo "  Platform: $$PLATFORM"; \
 	echo "  Template: $$TEMPLATE_FILE"
 
-es-upload:
+es-upload: es-template
```

checkmake (0.3.2)

[warning] Target body for "es-upload" exceeds allowed length of 5 lines (15).

(maxbodylength)